README.md

```diff
@@ -86,16 +86,16 @@ OPTIONS:
 
         --elasticsearch-user <USER>                   User to use for Elasticsearch logging [env: ELASTICSEARCH_USER=]
         --ethereum-ipc <NETWORK_NAME:FILE>
-            Ethereum network name (e.g. 'mainnet') and Ethereum IPC pipe, separated by a ':'
+            Ethereum network name (e.g. 'mainnet'), comma-seperated capabilities (eg `full,archive), and a Ethereum IPC pipe, separated by a ':'
```

Comment:
Thanks for the PR! The changes look good but there's a dangling backtick in this and the other places. Could we update that?

Also: the capabilities are optional, so it would be ideal to wrap them in `[CAPABILITIES:]` to make that clear.